### PR TITLE
Update from auto-update/renku-notebooks-1.19.0

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy extra-values=notebooks.userSessionPersistentVolumes.enabled=true

### CODE REVIEW

**Review by bethcg — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+  (`#1518 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1518>`_).
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**New Features**
+
+- **Renku Notebooks**: Use a new version of Amalthea which adds fields for culling
+  hibernating sessions in the CRD.
+- **Renku Notebooks**: ``notebook.culling.idleThresholdSeconds`` in the values file was renamed to
```
Comment (olevski) [inline CHANGELOG.rst:23]:
```suggestion
- **Renku Notebooks**: **Helm chart breaking change** ``notebook.culling.idleThresholdSeconds`` in the values file was renamed to
```

**Review by bethcg — APPROVED** [review]

**Comment by lorenzo-cavazzi** [comment]
This PR now contains all the changes for the persistent sessions

**Review by lokijuhy — APPROVED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -25,7 +64,7 @@ User-Facing Changes
 Individual components
 ~~~~~~~~~~~~~~~~~~~~~~
 
-- `renku-ui 3.12.21 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.12.2>`_
+- `renku-ui 3.12.2 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.12.2>`_
```
Comment (aledegano) [inline CHANGELOG.rst:67]: uhm, is this correct? seems something weird going on here...

Comment (aledegano) [reply CHANGELOG.rst:67]: seems like it should be `3.13.0` I think

Comment (olevski) [reply CHANGELOG.rst:67]: So this is correct.

**Review by aledegano — APPROVED** [review]

**Pull request merged**